package/python-kiwi_boxed_plugin-spec-template

```diff
@@ -73,7 +73,7 @@ Requires:       python%{python3_pkgversion}-docopt
 Requires:       python%{python3_pkgversion}-kiwi >= 9.21.21
 Requires:       python%{python3_pkgversion}-requests
 Requires:       python%{python3_pkgversion}-setuptools
-Requires:       python%{python3_pkgversion}-wget
+Requires:       python%{python3_pkgversion}-progressbar
```

Comment:
`progressbar2` is the maintained, API-compatible fork of `progressbar`. Use that instead.

Reply:
done